src/input.rs

```diff
@@ -218,27 +218,17 @@ const INPUT_FUNCTION_METADATA: &[InputFunctionMetadata] = &[
 ];
 assert_sorted_by_name!(INPUT_FUNCTION_METADATA);
 
-const fn _assert_sizes_match() {
-    let input_function_count = R_END_INPUT_FUNCTIONS;
-    assert!(
-        INPUT_FUNCTION_METADATA.len() == input_function_count,
-        concat!(
-            "input_function_metadata size mismatch with input_common. ",
-            "Did you forget to update input_function_metadata?"
-        )
-    );
-}
-const _: () = _assert_sizes_match();
-
-// Keep this function for debug purposes
-// See 031b265
-#[allow(dead_code)]
-pub fn describe_char(c: i32) -> WString {
-    if c > 0 && (c as usize) < R_END_INPUT_FUNCTIONS {
-        return sprintf!("%02x (%ls)", c, INPUT_FUNCTION_METADATA[c as usize].name);
-    }
-    return sprintf!("%02x", c);
-}
```

Comment:
Were these functions supposed to be removed?

Reply:
I didn't end up removing them